integration-test/src/test/scala/com/cloudera/livy/test/JobApiIT.scala

```diff
@@ -157,16 +157,17 @@ class JobApiIT extends BaseIntegrationTestSuite with BeforeAndAfterAll with Logg
     assert(result === "hello")
   }
 
-  test("run scala jobs") {
+  test("run echo jobs") {
```

Comment:
test-lib is picked up by code but not by classpath [Code](https://github.com/cloudera/livy/blob/master/integration-test/src/main/scala/com/cloudera/livy/test/framework/MiniCluster.scala#L227).
We can change it to use sparkScalaVersion to solve this problem.